scripts/zones/Xarcabard/mobs/Boreal_Hound.lua

```diff
@@ -15,6 +32,26 @@ entity.onMobSpawn = function(mob)
     end
 end
 
+entity.onMobRoam = function(mob)
+    local PathingIndex = mob:getLocalVar('PathingIndex')
+    local ResumePathingIndex = mob:getLocalVar('ResumePathingIndex')
+    
+    if mob:isFollowingPath() then
+        mob:setLocalVar('ResumePathingIndex', os.time() + 4) -- Make sure mob is waiting 4 seconds to path after stopping
+    elseif (os.time() > ResumePathingIndex) then
+        if (math.random() > .25) then
+            PathingIndex = PathingIndex + math.random(1,3)
```

Comment:
```lua
elseif os.time > resumePathingIndex then
    if math.random(1, 100) > 25 then
        pathingIndex = pathingIndex + math.random(1, 3)
```
- This is not c++. Lua doesnt need parenthesis wraping the conditions.
- Space after commas.
- Whenever possible, dont use math.random(nothing). Use concrete min and max

Reply:
why are we even randoming? retail is pretty consistent about what every step of the path including the directions faced give or take a little latency on when the updated positions reach the client via packet.